php_phongo_structs.h

```diff
@@ -81,7 +81,6 @@ typedef struct {
 typedef struct {
 	PHONGO_ZEND_OBJECT_PRE
 	mongoc_client_t    *client;
-	char               *pem_file;
```

Comment:
I forgot to remove this field in the PR for PHPC-605 when we dropped PHP streams.